vendor/plugins/edu.harvard.dce.paella.singleVideoToggle/singleVideoToggle.js

```diff
@@ -71,10 +108,13 @@ paella.addPlugin(function () {
           // toggle each source sequentially
           let index = This._toggleIndex++ % paella.dce.sources.length;
           paella.player.videoLoader._data.streams = [paella.dce.sources[index]];
+          This._printResIfHlsLive(paella.player.videoLoader._data.streams);
```

Comment:
I am not familiar with this code base, what object is the capitalized "This"?

Reply:
It's to save the scope to the class so class variables and methods can be used in promises. If you look higher, theres a " let This = this;" . Another common variable for this is "self". Some parts of the coder use "self" some "This" some "thisClass" etc. This code has